src/components/DateRangePicker.jsx

```diff
@@ -267,6 +269,15 @@ class DateRangePicker extends React.PureComponent {
     });
   }
 
+  onKeyDownTab(evt) {
+    const { focusedInput } = this.props;
+    if (focusedInput !== null) {
```

Comment:
There's another change (https://github.com/airbnb/react-dates/pull/1553) that's going to make it so that there's only one instance of the calendar, and it sits between the two inputs. This will end up breaking the behavior you're adding here, since it will cause a keyboard trap on the last input.

In general, it's not advised to manually handle focus from tabbing. What's the need for adding this behavior here?